pkg/dispatchcli/cmd/create.go

```diff
@@ -105,6 +119,11 @@ func importFile(out io.Writer, errOut io.Writer, cmd *cobra.Command, args []stri
 			if err != nil {
 				return errors.Wrapf(err, "Error decoding image document %s", string(doc))
 			}
+			manifest, err := resolveFileReference(m.RuntimeDependencies.Manifest)
```

Comment:
Is there an example of this somewhere?

Reply:
It's in my own demo, but I should probably add something to seed, so continues to work.